src/containers/SessionMenu.js

```diff
@@ -21,12 +23,29 @@ function addMockNodes() {
   * @extends Component
   */
 class SessionMenu extends Component {
-  componentWillMount() {
-    this.props.registerModal('EXPORT_DATA');
```

Comment:
So you don't need to register a modal to use it? Huh.

Reply:
Not anymore!  It's a recent change.

Reply:
It's been factored out into a HOC: https://github.com/codaco/Network-Canvas/blob/66a143abc915472ab158d5cd4c32ba0e314a1afc/src/behaviours/modal.js#L27